recipes/b-multi/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conan import ConanFile
+from conan.tools.files import copy
+
+
+class MultiConan(ConanFile):
+    name = "b-multi"
+    homepage = "https://gitlab.com/correaa/boost-multi"
+    description = "Multidimensional array access to contiguous or regularly contiguous memory. (Not an official Boost library)"
+    topics = (
+        "array",
+        "multidimensional",
+        "library",
+    )
+    license = "Boost"
+    url = "https://gitlab.com/correaa/boost-multi"
+    # No settings/options are necessary, this is header only
+    exports_sources = "include/*"
+    no_copy_source = True
+
+    def package(self):
+        # This will also copy the "include" folder
+        copy(self, "*.hpp", self.source_folder, self.package_folder)
```

Comment:
```suggestion
        copy(self, "*.hpp", os.path.join(self.source_folder, "include"), os.path.join(self.package_folder, "include"))
```
Avoid copying unnecessary files in the future, like from test and example folders.

Reply:
I see now where `include/` appears.